plugins/services/src/js/structs/ServiceTree.js

```diff
@@ -119,7 +119,10 @@ module.exports = class ServiceTree extends Tree {
    */
   findServiceByName(name) {
     return this.findItem(function(item) {
-      return item.getName() === name && item instanceof Service;
+      return (
+        (item instanceof Framework && item.getFrameworkName() === name) ||
```

Comment:
Since this method is only being used once in `MesosStateStore` and only needs the case where we find by `getFrameworkName`, I think the former behavior should be completely removed.